CHANGELOG.md

```diff
@@ -3,8 +3,16 @@ Changelog for NeoFS Monitor
 
 ## [Unreleased]
 
+## [0.12.1] - 2024-09-03
+
 ### Changed
-- Go 1.21+ is required to build now (#114)
+- Go 1.22+ is required to build now (#149, #150)
+- Dropped 'v' prefix from version metric (#151)
+- Updated github.com/nspcc-dev/locode-db dependency to 0.7.0 (#150)
```

Comment:
imo, the whole "update" changes list should be placed to a separate section (in the node it is Update)

Reply:
This chlog almost doesn't use it (well, https://github.com/nspcc-dev/.github/issues/3).